client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditTable.tsx

```diff
@@ -36,15 +39,24 @@ const TotalCell = styled(TableCell)({
   fontWeight: 'bold',
 });
 
-const PlaceholderRow = ({ quantity }: { quantity: number | null }) => {
+const PlaceholderRow = ({
+  quantity,
+  extraColumnOffset,
+}: {
+  quantity: number | null;
+  extraColumnOffset: number;
+}) => {
   const t = useTranslation();
 
   const formattedValue = useFormatNumber().round(quantity ?? 0, 2);
 
   // TODO - maybe should be editable? Can't clear when manually allocating..
   return quantity === null ? null : (
     <tr>
-      <PlaceholderCell colSpan={3} sx={{ color: 'secondary.main' }}>
+      <PlaceholderCell
+        colSpan={4 + extraColumnOffset}
```

Comment:
we're added a bunch of extra columns to the outbound edit modal... this was best way I could think to bump the placeholder/total values over enough. We'll just add to `extraColumnOffset` below based on prefs? Would be better if these values actually just rendered themselves inside the defined columns though? Maybe better as a refactor issue...